cropharvest/datasets.py

```diff
@@ -83,24 +84,19 @@ def __init__(self, root, download=False):
         super().__init__(root, download, filenames=(LABELS_FILENAME,))
 
         # self._labels will always contain the original dataframe;
-        # the CropHarvestLabels class should not modify it
-        self._labels = read_geopandas(self.root / LABELS_FILENAME)
+        # the CropHarvestLabels class should not modify it.
+        # We use the Engineer load_labels since this also loads the .h5
+        # filenames for each row
+        # TODO: The load_labels doesn't actually allow the root to be
+        # modified. We should probably do this at a package level, not
+        # at a class level
+        self._labels = Engineer.load_labels(root=root)
```

Comment:
@ivanzvonkov , do you think the idea of a `root` is still relevant? Or is the data is small enough that we can just have the `DATAFOLDER_PATH` act as the root and remove that as an option for the user?

Reply:
per discussion today I think removing is fine